internal/rules/policies/kubernetes/policies/aquacommercial/service_with_externalip_test.rego

```diff
@@ -0,0 +1,41 @@
+package builtin.kubernetes.KSV0108
+
+test_service_with_externalip_denied {
+	r := deny with input as {
```

Comment:
Example for service with external ip
![image](https://user-images.githubusercontent.com/50172913/201108786-69398e27-1d6b-492d-a85c-c55c832c71e9.png)